bin/profiles/default.sh

```diff
@@ -87,6 +87,10 @@ fi
 echo "Done."
 echo "Building Acquia Content Hub from branch '${ACH_BRANCH}'"
 
+# Installing Acquia Content Hub Console
+COMPOSER_MEMORY_LIMIT=-1 composer config repositories.acquia_console '{"type":"vcs","url":"git@github.com:acquia/console.git","no-api":true}'
+COMPOSER_MEMORY_LIMIT=-1 composer require acquia/console:'0.0.x-dev'
```

Comment:
This command did not work for me.
```
COMPOSER_MEMORY_LIMIT=-1 composer require acquia/console:'0.0.x-dev'                                         2 ↵  3160  16:10:12


  [InvalidArgumentException]
  Could not find package acquia/console in a version matching 0.0.x-dev

```
It works without specifying version, like:
`COMPOSER_MEMORY_LIMIT=-1 composer require acquia/console`

Reply:
Thanks :)